downstream/modules/platform/proc-central-auth-install-dependencies.adoc

```diff
@@ -0,0 +1,46 @@
+[id="proc-central-auth-dependencies"]
+
+= Install the collections and dependencies for Ansible Galaxy
+
+.Prerequisites
+
+. A supported version of either the https://access.redhat.com/product-life-cycles[Red Hat Enterprise Linux] or the https://docs.fedoraproject.org/en-US/releases/[Fedora] operating system
+
+. A supported version of https://access.redhat.com/support/policy/updates/ansible-automation-platform[Ansible]
```

Comment:
of link:https://access.redhat.com/support/policy/updates/ansible-automation-platform[{PlatformNameShort}]